10up/classifai
# Fix: Generate/regenerate/rescan buttons do not work in media modal when editing a post

### Description of the Change

The buttons in the media only work under the Media library page. This happens because we only listen to `edit:attachment` event triggered when you open the media modal of an image on the Media library page. This PR adds a listener to another event to make the buttons work when editing a post.


### Verification Process

1. Build new assets.
2. Edit a post using the block editor, see buttons work.
3. Edit a post using the classic editor, see buttons work.
4. Edit an attachment, see buttons work.



### Checklist:






### Applicable Issues
Close #292.


### Changelog Entry

Fixed: Ensure Image Processing buttons in the media modal work when editing posts.
